mmj2jar/LinuxMMJ2PATutorial.command

```diff
@@ -0,0 +1 @@
+java -XX:+UseG1GC -Xms128M -Xmx1024M -jar /users/Userone/mmj2jar/mmj2.jar "LinuxRunParmsPATutorial.txt" Y "/users/Userone/mmj2jar/" "/users/Userone/metamath/" "" 

```

Comment:
What is `/users/Userone`? Perhaps this should be `~` instead. Besides this, if one of these files is identical to one from another system (I think `LinuxRunParmsPATutorial.txt` has nothing linux-specific in it), we should just redirect people to the other file.